### DISCUSSION TITLE
.net9 support?

### DISCUSSION THREAD

**Discussion by ggbtvs** [MvvmCross/MvvmCross#4858] [General]
Mvvmcross supporting .net9. Is this on the roadmap at all?

**Comment by entdark** [comment]
It's not even released yet.

**Comment by Cheesebaron** [comment]
Probably not much effort to support it. So probably yes.

**Reply by ggbtvs** [reply]
Will I still be able to use the .net 8 version of Mvvmcross in .net 9 iOS app?

**Reply by Cheesebaron** [reply]
Yes

**Reply by Cheesebaron** [reply]
https://github.com/MvvmCross/MvvmCross/pull/4869
